index.html

```diff
@@ -1,28 +1,196 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-
-    <title>Hello!</title>
-    
-    <!-- import the webpage's stylesheet -->
-    <link rel="stylesheet" href="/style.css">
-    
-    <!-- import the webpage's javascript file -->
-    <script src="/script.js" defer></script>
-  </head>  
-  <body>
-    <h1>Hi there!</h1>
-    
-    <p>
-      I'm your cool new webpage. Made with <a href="https://glitch.com">Glitch</a>!
-    </p>
-
-    <!-- include the Glitch button to show what the webpage is about and
-          to make it easier for folks to view source and remix -->
-    <div class="glitchButton" style="position:fixed;top:20px;right:20px;"></div>
-    <script src="https://button.glitch.me/button.js" defer></script>
-  </body>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Chulmin Han P1 Resume</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@300;600&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <div class="container">
+
+
+        <!-------------------
+         PERSONAL-INFORMATION
+        --------------------->
+
+        <div class="p-info">
+            <h1>
+                Chulmin Han
+            </h1>
+
+            <p class="address">
+                Dallas, Texas
+            </p>
+
+            <p class="c-info">
+                chan01@mica.edu / 214-773-8296
```

Comment:
For an HTML resume, it also makes sense for your email address to be a link; you do that by using [`mailto`](https://css-tricks.com/snippets/html/mailto-links/):
```suggestion
                <a href="mailto:chan01@mica.edu">chan01@mica.edu</a> / 214-773-8296
```